src/main/groovy/org/transmartproject/batch/concept/ConceptNode.groovy

```diff
@@ -35,6 +35,6 @@ class ConceptNode {
     }
 
     boolean isNew() {
-        code == null
+        i2b2RecordId == null
     }
```

Comment:
Fair enough, but then it should be said in the code. It would also be a good idea to add an assertion to the code loading concepts from the database that `record_id` is always filled.